packages/devtools_app/lib/src/debugger/debugger_controller.dart

```diff
@@ -732,6 +733,76 @@ class DebuggerController extends DisposableController
     return boundsVariables.map((bv) => Variable.create(bv)).toList();
   }
 
+  // Decodes the bytes into the correctly sized values based on the instance
+  // type, falling back to raw bytes. The switch is based on the allowed
+  // values of `instance.bytes`.
+  List<Variable> _createVariablesForBytes(Instance instance) {
+    final bytes = base64.decode(instance.bytes);
+    final boundVariables = <BoundVariable>[];
+    List<dynamic> result;
+    switch (instance.kind) {
+      case 'Uint8ClampedList':
+      case 'Uint8List':
+        result = bytes;
+        break;
+      case 'Uint16List':
+        result = Uint16List.view(bytes.buffer);
+        break;
+      case 'Uint32List':
+        result = Uint32List.view(bytes.buffer);
+        break;
+      case 'Uint64List':
+        result = Uint64List.view(bytes.buffer);
+        break;
+      case 'Int8List':
+        result = Int8List.view(bytes.buffer);
+        break;
+      case 'Int16List':
+        result = Int16List.view(bytes.buffer);
+        break;
+      case 'Int32List':
+        result = Int32List.view(bytes.buffer);
+        break;
+      case 'Int64List':
+        // TODO: The Int64List cannot be instantiated on the web. Consider
```

Comment:
Could pull in fixnum, wasn't sure if there was an existing pattern to handle displaying ints that are outside of javascript's safe range